nilearn/regions/signal_extraction.py

```diff
@@ -397,6 +402,18 @@ def img_to_signals_maps(imgs, maps_img, mask_img=None):
         Every point outside the mask is considered
         as background (i.e. outside of any region).
 
+    keep_masked_maps : :obj:`bool`, optional
+        If True, masked atlas with invalid maps (maps with no brain coverage
+        after applying the mask) will be retained in the output, resulting
+        in corresponding time series containing zeros only. If False, the
```

Comment:
if you want you can use the `_utils.fill_doc` to avoid having to update the docstring in several places